libr/include/r_util/r_alloc.h

```diff
@@ -52,6 +52,22 @@ R_API void r_free(void *p);
 #define _r_free r_free
 #define _r_realloc r_realloc
 
+static inline void *r_malloc_logerr (size_t n) {
```

Comment:
`r_malloc_check` sounds better honestly
but it could make monkey think that the program would abort on malloc fail

Reply:
That sounds like a good addition for 5.8, because i was planning to make malloc calls configurable at compile time, to allow linking r2 with custom allocators. But we can extend this thing to also avoid null checks on malloc failure or print a message or just return. Just like we have for rreturnvaliffail

Reply:
well this pr is fine to merge because its burning lots of eprintfs already. but yeah, hold those malloc changes for 3.7.9